carmirror/core.go

```diff
@@ -357,6 +369,16 @@ func NewReceiverSession[I BlockId, F Flags](
 	}
 }
 
+// Provide access to orchestrator
```

Comment:
```suggestion
// Orchestrator provides access to the orchestrator.
```

Just matching Go's doc recommendations.